Add Fleet & Agent 8.10.3 Release Notes (#552)
elastic/ingest-docs: docs/en/ingest-management/release-notes/release-notes-8.10.asciidoc

* {kibana-ref}/release-notes.html[{kib} release notes]
* {beats-ref}/release-notes.html[{beats} release notes]

// begin 8.10.3 relnotes

[[release-notes-8.10.3]]
== {fleet} and {agent} 8.10.3

Review important information about the {fleet} and {agent} 8.10.3 release.

[discrete]
[[known-issues-8.10.3]]
=== Known issues

IMPORTANT: The <<known-issue-3375-v8100,known issue>> that prevents successful upgrades in an air-gapped environment for {agent} versions 8.9.0 to 8.10.2 has been resolved in this release. If you're using an air-gapped environment, we recommend installing version 8.10.3 or any higher version to avoid not being unable to upgrade.

[discrete]
[[enhancements-8.10.3]]
=== Enhancements

{agent}::

> * Improve {agent} uninstall by adding some pause between retries when removal is blocked by busy files {agent-pull}3431[#3431] {agent-issue}3221[#3221]

Reviewer: ```suggestion
* Improve {agent} uninstall on Windows by adding delay between retries when file removal is blocked by busy files {agent-pull}3431[#3431] {agent-issue}3221[#3221]
```

Let's note that this one is Windows specific.

[discrete]
[[bug-fixes-8.10.3]]
=== Bug fixes

{fleet}::
* Fix incorrect index template used from the data stream name ({kibana-pull}166941[#166941])
* Increase package install max timeout limit and add concurrency control to rollovers ({kibana-pull}166775[#166775])
* Fix bulk action dropdown ({kibana-pull}166475[#166475])

{agent}::

> * Fix GPG verification; if one GPG check is successful the upgrade should continue. {agent-pull}3426[#3426] {agent-issue}3368[#3368]
> * Resilient handling of air gapped PGP checks. {agent} should not fail when remote PGP is specified (or official Elastic fallback PGP is used) and remote is not available {agent-pull}3427[#3427] {agent-issue}3368[#3368]

Reviewer: ```suggestion
* Resilient handling of air gapped PGP checks. {agent} should not fail when remote PGP is specified (or official Elastic fallback PGP is used) and remote is not available {agent-pull}3427[#3427] {agent-pull}3426[#3426] {agent-issue}3368[#3368]
```

Let's combine these two into one entry, they are both solving the same problem for the same issue.
